applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj.conf

```diff
@@ -106,3 +106,5 @@ CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
 
 CONFIG_DESKTOP_LOG=y
+
+CONFIG_NRFX_GPPI=y
```

Comment:
This shouldn't be necessary as mentioned in https://github.com/nrfconnect/sdk-nrf/pull/24694#issuecomment-3362319289
Should be added for example here: https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/mpsl/fem/Kconfig#L83-L85 and https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/mpsl/pin_debug/Kconfig#L32-L33

Reply:
Please also add it for mpsl_pin_debug